aes/src/armv8.rs

```diff
@@ -235,6 +249,17 @@ macro_rules! define_aes_impl {
             }
         }
 
+        #[cfg(feature = "zeroize")]
+        impl Drop for $name_dec {
+            fn drop(&mut self) {
+                use cipher::zeroize::Zeroize as _;
+                self.round_keys.zeroize();
+            }
+        }
```

Comment:
I think it might be better to define `Drop` unconditionally and gate zeroization in the body:

```suggestion
        impl Drop for $name_dec {
            fn drop(&mut self) {
                #[cfg(feature = "zeroize")]
                zeroize::Zeroize::zeroize(&mut self.round_keys);
            }
        }
```

The same goes for the other impls.

`Drop` is a bit magical so conditionally defining it seems a bit weird. It also changes the bounds.